server/Dockerfile

```diff
@@ -137,6 +137,8 @@ ARG PHP_VER
     RUN if [ ! -z ${MISP_COMMIT} ]; then cd /var/www/MISP && git checkout ${MISP_COMMIT}; fi; \
         # We build the MISP modules outside, so we don't need to grab those submodules
         cd /var/www/MISP/app || exit; git submodule update --init --recursive .;
+    # Remove some old and broken links that pollute the log files
+    RUN rm -rf /var/www/MISP/INSTALL/old
```

Comment:
Can we inline in the `RUN` above so we don't create an additional layer?

Reply:
ACK